### PR TITLE
add dumme comands

### PR DESCRIPTION
**Pull request by UrUrZa**
add dumme teacher commands

### CODE REVIEW

File: src/teacher/commands/domain/domain.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+);
+
+export const changeAddParticipantSearch		= createCommand<string>(
+  'changeAddParticipantSearch',
+  function*({payload: participantSearch}) {
+    yield call(console.log,'requestStatusChange', participantSearch);
+  }
+);
+
+export const confirmAddParticipantSearch = createCommand<void>(
```
Comment (Kirill486) [inline src/teacher/commands/domain/domain.ts:68]: confirmAddPrticipant

File: src/teacher/commands/domain/domain.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  'confirmAddParticipantSearch',
+  function*() {
+    yield call(console.log,'confirmAddParticipantSearch');
+  }
+);
+
+export const startTeacerApp = createCommand<void>(
```
Comment (Kirill486) [inline src/teacher/commands/domain/domain.ts:75]: teacher

File: src/teacher/commands/editable/editable.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+export const changeRuleStartTime	= createCommand<Date>(
+  'changeRuleStartTime',
+  function*({payload: startTime}) {
+    yield call(console.log,'changeRuleStartTime', startTime);
+  }
+);
+
+export const changeRuleDuration	= createCommand<number>(
+  'changeRuleDuration',
+  function*({payload: duration}) {
```
Comment (Kirill486) [inline src/teacher/commands/editable/editable.ts:119]: `durationMin`

File: src/teacher/commands/editable/editable.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+export const changeRuleMonthDay	= createCommand<Date>(
+  'changeRuleMonthDay',
+  function*({payload: monthDay}) {
+    yield call(console.log,'changeRuleMonthDay', monthDay);
+  }
+);
+
+export const changeAbonementSubjects	= createCommand<id[]>(
+  'changeAbonementSubjects',
+  function*({payload: subjectIdArray}) {
```
Comment (Kirill486) [inline src/teacher/commands/editable/editable.ts:133]: `subjectsIdArray`

File: src/teacher/commands/editable/editable.ts [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    yield call(console.log,'changeAbonementSubjects', subjectIdArray);
+  }
+);
+
+export const deleteAbonementSubject	= createCommand<id[]>(
```
Comment (Kirill486) [inline src/teacher/commands/editable/editable.ts:138]: `id[]`?

File: src/teacher/commands/novigation/navigation.ts [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+);
+
+export const teacherShowMore = createCommand<void>(
+  'teacherShowMore',
+  function*() {
+    yield call(console.log,'teacherShowMore');
+  }
+);
+
+export const startChangeSubjectDescription = createCommand<void>(
```
Comment (Kirill486) [inline src/teacher/commands/novigation/navigation.ts:110]: this seem to be an `editables` command

File: src/teacher/commands/novigation/navigation.ts [hunk 7]
```diff
@@ -0,0 +1,199 @@
+import { call } from "redux-saga/effects";
+import { id } from "../../../common/types/domain";
+import { createCommand } from "../../../common/utilities/createCommand";
+
+export const setTeacherScheduleView	= createCommand<void>(
```
Comment (Kirill486) [inline src/teacher/commands/novigation/navigation.ts:5]: where are other teacher views?

**Review by Kirill486 — CHANGES_REQUESTED** [review]
Its better to split commits. A single one is not the best practice.

I request changes.

File: src/teacher/commands/domain/domain.ts (hunk 1 above)
Comment (UrUrZa) [reply src/teacher/commands/domain/domain.ts:68]: `searchParticipantToAdd`

File: src/teacher/commands/novigation/navigation.ts (hunk 7 above)
Comment (UrUrZa) [reply src/teacher/commands/novigation/navigation.ts:5]: resolved

**Pull request merged**